ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/KeyValueSubmission.scala

```diff
@@ -78,7 +78,7 @@ class KeyValueSubmission(metrics: Metrics) {
         .addAllInputDamlState(contractKeyStates.asJava)
         .setTransactionEntry(
           DamlTransactionEntry.newBuilder
-            .setTransaction(Conversions.encodeTransaction(tx))
+            .setRawTransaction(Conversions.encodeTransaction(tx).toByteString)
```

Comment:
```suggestion
            .setRawTransaction(Conversions.encodeTransaction(tx).bytes)
```

assuming that the type of `encodeTransaction` as been changed as stated before.